src/Components/Button/button.scss

```
@@ -0,0 +1,14 @@
+.button {
+  height: 40px;
+  border-width: 0px;
+  color: white;
+  font-weight: bold;
+  padding-inline: 30px;
+  justify-self: center;
+
+  &--news {

```

```suggestion
  &--primary {

```
You can call this as the primary button style, inspired by https://getbootstrap.com/docs/5.1/components/buttons/#examples